packages/gatsby-theme-newrelic/src/components/Banner.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { css } from '@emotion/core';
+import Button from './Button';
+import Icon from './Icon';
+
+const Banner = ({ children, color }) => {
+  const [visible, updateVisible] = useState(true);
+
+  if (!visible) return null;
```

Comment:
```suggestion
  if (!visible) {
    return null;
  }
```

[nit] Could we use brackets around `if` statements? Maybe I'm the minority here, but I find it a bit hard to follow without the brackets. The brackets give me a super quick way to understand what is grouped inside the condition with a bit less congitive overhead. Thoughts?